src/pages/Curation/components/PresentRecommendation/index.tsx

```diff
@@ -65,25 +64,54 @@ export type ProductType = {
 const PresentRecommendation = () => {
   const locationData = useLocation();
   const navigate = useNavigate();
+  const [storageData, setStorageData] = useState<any>('');
 
   const handleGoBack = () => {
     window.history.back();
   };
 
-  const [maxPercentage, setMaxPercentage] = useState<number>();
-  const [tags, setTags] = useState([]);
+  let maxPercentage = localStorage.getItem('states')
+    ? storageData?.maxPercentage
+    : locationData?.state[1];
+  let tags = localStorage.getItem('states') ? storageData?.tags : locationData?.state[2];
```

Comment:
타입을 알 수 있다면 명시해주면 좋을 것 같습니다.
```suggestion
  let maxPercentage: 타입 = localStorage.getItem('states')
    ? storageData?.maxPercentage
    : locationData?.state[1];
  let tags: 타입 = localStorage.getItem('states') ? storageData?.tags : locationData?.state[2];
```